services/director-v2/tests/unit/with_dbs/test_api_route_dynamic_services.py

```diff
@@ -491,15 +496,15 @@ def test_delete_service_waiting_for_manual_intervention(
 
     # mark service as failed and waiting for human intervention
     node_uuid = UUID(service["node_uuid"])
-    scheduler_data = dynamic_sidecar_scheduler._scheduler.get_scheduler_data(  # pylint: disable=protected-access
+    scheduler_data = dynamic_sidecar_scheduler._scheduler.get_scheduler_data(  # pylint: disable=protected-access  # noqa: SLF001
         node_uuid
     )
     scheduler_data.dynamic_sidecar.status.update_failing_status("failed")
     scheduler_data.dynamic_sidecar.wait_for_manual_intervention_after_error = True
 
     # check response
     url = URL(f"/v2/dynamic_services/{node_uuid}")
-    stop_response = client.delete(str(url), allow_redirects=False)
+    stop_response = client.delete(str(url), follow_redirects=False)
```

Comment:
aha! so this was the problem? the name was wrong?

Reply:
no this was deprecated. the problem was different.